src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import { maskSSN } from '../../../../utils/helpers/general';
+
+describe('hca `maskSSN` method', () => {
```

Comment:
Any concerns around passing a string that is 4 characters or less? I know we probably won't use it except for when we have a full ssn, but what would we expect the `maskSSN` to return for example if it was passed `333`?

Reply:
We've never had one come across short on character count or formatted incorrectly. It's always been an all-or-nothing-type thing. I supposed we could try to pass one that isn't formatted correctly and see what happens.